docs/DeveloperGuide.md

```diff
@@ -424,51 +431,58 @@ Extensions:
 
     Use case ends.
 
-**Use case: UC02 - Delete a student profile**
+**Use case: UC0X - Find a student’s profile**
 
 MSS:
 
-1. User enters the delete student command, along with the student’s name.
-2. TutorBuddy verifies that the student profile exists.
-3. TutorBuddy prompts the user to confirm the deletion.
-4. User confirms the deletion.
+1. User enters the `find_student` command, along with a keyword from the student’s name.
+2. TutorBuddy displays all students’ profiles matching the keyword if any.
 
    Use case ends.
 
 Extensions:
 
-* 2a. TutorBuddy detects an error in the entered command.
-    * 2a1. TutorBuddy displays error messages to the user.
+* 1a. TutorBuddy detects empty keyword field
+    * 1a1. TutorBuddy displays an error message for no keyword specified.
 
   Use case ends.
 
-* 3a. TutorBuddy detects that the user does not exist.
-    * 3a1. TutorBuddy displays an error message for unknown student profiles.
+**Use case: UC0X - Delete a student profile**
 
-  Use case ends.
+MSS:
+
+1. User enters the `delete_student` command, along with the student’s name.
+2. TutorBuddy verifies that the inputs are valid and student profile exists.
+3. TutorBuddy displays a success message.
 
-* 4a. User cancels the confirmation of deletion.
-    * 4a1. Student profile is not created; the student is returned to the home page.
+   Use case ends.
+
+Extensions:
+
+* 2a. TutorBuddy detects and error in the input.
```

Comment:
an error

Reply:
i think similar accross

Reply:
Nice catch! I have updated it in the latest commit.